src/components/routeComponents/memberRegistrationCorrectionForm/AssignLevelAndClassroomModal.jsx

```diff
@@ -0,0 +1,127 @@
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import fields from 'components/common/fields';
+import Form from 'pepcus-core/lib/Form';
+import Row from 'pepcus-core/lib/Row';
+import InfoMessage from '../../common/InfoMessage';
+import Col from 'pepcus-core/lib/Col';
+import { useState } from 'react';
+import Modal from 'pepcus-core/lib/Modal';
+import Button from 'pepcus-core/lib/Button';
+
+const StyledInfoContainer = styled.div`
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	.fsOZUI {
+		width: 100%;
+		margin: 0;
+	}
+`;
+
+const StyledForm = styled.div`
+	width: 100%;
+	.ebPCkL {
+		margin-bottom: 0 !important;
+	}
+`;
+
+const StyledRow = styled(Row)`
+	font-size: 22px;
+	color: #5864fb;
+	padding: 0 200px;
+	@media (max-width: 767px) {
+		padding: 0;
+		font-size: 18px;
+	}
+`;
+
+const StyledCol = styled(Col)`
+	flex-basis: 11%;
+`;
+
+const AssignLevelAndClassroomModal = ({
+	constants,
+	config,
+	formData,
+	onChange,
+	renderCloseModal,
+	renderSubmitButtons,
+}) => {
+	const { ASSIGN_CLASSROOM, CLOSE, SUBMIT } = constants;
+	const { assignLevelAndClassroomConfig } = config;
+	const [isOpenModel, setIsOpenModal] = useState(true);
+	const closeModal = () => {
+		setIsOpenModal(false);
+		renderCloseModal();
+	};
+
+	return (
+		<Modal
+			open={isOpenModel}
+			onClose={closeModal}
+			style={{ padding: '22px 16px', borderRadius: '10px' }}>
+			<StyledRow justify="center" margin="0 0 25px 0">
+				{ASSIGN_CLASSROOM}
+			</StyledRow>
+			<Row justify="center" margin="25px 0">
+				<StyledForm>
+					<Form
+						externalSubmission
+						fields={fields}
+						liveValidate
+						schema={assignLevelAndClassroomConfig.schema}
+						showErrorList={false}
+						formData={formData}
+						onChange={onChange}
+						uiSchema={assignLevelAndClassroomConfig.uiSchema}
+					/>
+				</StyledForm>
+			</Row>
+			{!formData.classAttended && (
+				<StyledInfoContainer>
+					<InfoMessage message={'Please select level and fetch classroom.'} />
```

Comment:
Get the message string from json constants 
```suggestion
					<InfoMessage message={SELECT_LEVEL_AND_CLASSROOM_MSG} />
```